src/App/Resources/AppResources.resx

```diff
@@ -2491,4 +2491,22 @@ Do you want to switch to this account?</value>
   <data name="LogInWithAnotherDevice" xml:space="preserve">
     <value>Log In with another device</value>
   </data>
+  <data name="LogInInitiated" xml:space="preserve">
+    <value>Log in initiated</value>
+  </data>
+  <data name="ANotificationHasBeenSentToYourDevice" xml:space="preserve">
+    <value>A notification has been sent to your device.</value>
+  </data>
+  <data name="PleaseMakeSureYourVaultIsUnlockedAnd" xml:space="preserve">
+    <value>Please make sure your vault is unlocked and the Fingerprint phrase matches on the other device.</value>
+  </data>
```

Comment:
It's not that long to make the key like the value, I'd do so. Also, by reading the key is like your lacking something or you don't know if it needs a parameter or something else.